### PR TITLE
[sw/silicon_creator] Update OWNER_SW_CFG_ROM_ALERT_DIGEST_RMA

### PR DESCRIPTION
**Pull request by alphan**
This PR updates the alert handler config digest in RMA. This should have been handled in #14639 but was masked because of a CI issue.

We are still working on the tooling to generate these values automatically but for now we can use this [workaround](https://gist.github.com/alphan/58d3e3dcba9f10837925389ff91b8d49).

Signed-off-by: Alphan Ulusoy <alphan@google.com>

### CODE REVIEW

**Review by milesdai — APPROVED** [review]

**Review by alphan — COMMENTED** [review]
.

**Comment by alphan** [comment]
@tjaychen this PR fixes the alert handler config digests but the remaining failures don't seem to be related to it:
```
//sw/device/tests:rstmgr_alert_info_test_fpga_cw310                      FAILED in 2 out of 2 in 3.3s
  Stats over 2 runs: max = 3.3s, min = 3.3s, avg = 3.3s, dev = 0.0s
  /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/rstmgr_alert_info_test_fpga_cw310/test.log
  /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/rstmgr_alert_info_test_fpga_cw310/test_attempts/attempt_1.log
//sw/device/tests/autogen:alert_test_fpga_cw310                          FAILED in 2 out of 2 in 18.9s
  Stats over 2 runs: max = 18.9s, min = 3.4s, avg = 11.1s, dev = 7.7s
  /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/autogen/alert_test_fpga_cw310/test.log
  /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/autogen/alert_test_fpga_cw310/test_attempts/attempt_1.log
```

**Review by dmcardle — APPROVED** [review]
LGTM, but could you add more detail on how you produced the new value?

**Comment by alphan** [comment]
> LGTM, but could you add more detail on how you produced the new value?

Sure, for now can you take a look at #14732 14038

**Review by vogelpi — APPROVED** [review]
LGTM, this should unblock CI.

Please make sure CI indeed passes before merging this.

**Review by a-will — APPROVED** [review]

**Comment by alphan** [comment]
> LGTM, this should unblock CI.
> 
> Please make sure CI indeed passes before merging this.

There are 2 remaining cw310 failures but those should be fixed separately.

**Comment by tjaychen** [comment]
> @tjaychen this PR fixes the alert handler config digests but the remaining failures don't seem to be related to it:
> 
> ```
> //sw/device/tests:rstmgr_alert_info_test_fpga_cw310                      FAILED in 2 out of 2 in 3.3s
>   Stats over 2 runs: max = 3.3s, min = 3.3s, avg = 3.3s, dev = 0.0s
>   /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/rstmgr_alert_info_test_fpga_cw310/test.log
>   /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/rstmgr_alert_info_test_fpga_cw310/test_attempts/attempt_1.log
> //sw/device/tests/autogen:alert_test_fpga_cw310                          FAILED in 2 out of 2 in 18.9s
>   Stats over 2 runs: max = 18.9s, min = 3.4s, avg = 11.1s, dev = 7.7s
>   /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/autogen/alert_test_fpga_cw310/test.log
>   /root/.cache/bazel/_bazel_root/5e4f8c833cf3a422fdf6ca333dd151d7/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/autogen/alert_test_fpga_cw310/test_attempts/attempt_1.log
> ```

sounds good, thank you @alphan

**Pull request merged**
